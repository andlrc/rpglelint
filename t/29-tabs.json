[
   {
      "column":8,
      "file":"t/29-tabs.rpgle",
      "line":2,
      "message":"global declaration 'key' is not allowed",
      "severity":"warning"
   }
]
