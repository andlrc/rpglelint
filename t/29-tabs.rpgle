// rpglelint: -Wglobal
	dcl-s key int(10);
