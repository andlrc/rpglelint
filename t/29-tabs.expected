t/29-tabs.rpgle:2:8: warning: global declaration 'key' is not allowed [-Wglobal]
  dcl-s key int(10);
        ^
